src/config/jwt.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  /*
+   *--------------------------------------------------------------------------
+   * JWT Secret
+   *--------------------------------------------------------------------------
+   *
+   * Here your jwt secret key. In order to verify a token injected in
+   * "Authorization" header, we should have way to decrypt that token
+   * and validate things.
+   *
+   */
+
+  secret: 'YOUR-JWT-TOKEN-HERE',
```

Comment:
Should be configurable per environment. i.e;

```
process.env.JWT_SECRET=
```

Reply:
Above is changed